cashu/core/base.py

```diff
@@ -161,18 +161,20 @@ class Proof(BaseModel):
 
     id: Union[
         None, str
-    ] = ""  # NOTE: None for backwards compatibility for old clients that do not include the keyset id < 0.3
+    ] = (  # NOTE: None for backwards compatibility for old clients that do not include the keyset id < 0.3
+        ""
+    )
```

Comment:
Weird but OK, assume `ruff` did this... do you know which rule/style by any chance? Don't think I've seen 'scoped' comments before

Reply:
TBF I'd prefer the comment to move a line up rather than be put in to brackets with the thing it's a comment about, but meh -- i.e. this seems clearer:

```python
# NOTE: None for backwards compatibility for old clients that do not include the keyset id < 0.
id: Union[None, str] = ""
```

Reply:
yep i think those inline same line comments are bad practice in general especially when they are quite long. black cant handle it :).